src/test/javascript/spec/component/course/course-exercises.component.spec.ts

```diff
@@ -290,4 +296,71 @@ describe('CourseExercisesComponent', () => {
         expect(component.activeFilters).toEqual(new Set());
         expect(component.numberOfExercises).toBe(4);
     });
+
+    it('should not filter exercises with an individual due date after the current date', () => {
+        component.sortingOrder = ExerciseSortingOrder.DESC;
+
+        // regular due date is in the past, but the individual one in the future
+        const newExercise = new ModelingExercise(UMLDiagramType.ClassDiagram, course, undefined) as Exercise;
+        newExercise.dueDate = dayjs('2021-01-13T16:11:00+01:00');
+        newExercise.releaseDate = dayjs('2021-01-13T16:11:00+01:00');
+        const participation = new StudentParticipation();
+        participation.individualDueDate = dayjs().add(3, 'days');
+        newExercise.studentParticipations = [participation];
+
+        component.course!.exercises![1] = newExercise;
+
+        component.activeFilters.clear();
+        component.toggleFilters([ExerciseFilter.OVERDUE]);
+
+        // ToDo: check that exercise is shown
```

Comment:
Is this to do left for a later PR or you just forgot to remove it?

Reply:
Good catch, I just forgot about it. Fixed in 9750ee4.